src/services/subsocial/profiles/fetcher.ts

```diff
@@ -37,6 +33,11 @@ const GET_PROFILES = gql`
         id
         name
         image
+        about
+        email
+        linksOriginal
+        tagsOriginal
```

Comment:
Do you use the data?

Reply:
its needed because if I don't get the data, when I update the profile (change username)
those unincluded data will be gone